apps/sms/js/drafts.js

```diff
@@ -265,8 +173,7 @@
         });
         draftIndex = new Map(records || []);
 
-        // Return list of threadless drafts
-        deferredDraftRequest.resolve(Drafts.byThreadId(null));
+        deferredDraftRequest.resolve();
```

Comment:
no real opinion here; you say "we won't need to expose getAll publicly" but if `request` resolves to the generator then `request` is essentially like `getAll` :) so in the end... I don't really care ;)